doc/developers.md

```diff
@@ -73,11 +73,11 @@ git push origin SPARKC-9999-b3.1
 # Repeat for b3.2
```

Comment:
This should be `Repeat for b3.2, b3.3, b3.4`

Reply:
fixed